packages/blocks/image/src/Image.tsx

```diff
@@ -329,12 +331,12 @@ export const Image: BlockComponent<AppProps> = (props) => {
     });
   };
 
-  if (stateObject.src?.trim()) {
+  if (stateObject.src?.trim() || url) {
     return (
       <div className={tw`flex justify-center text-center w-full`}>
         <div className={tw`flex flex-col`}>
           <ResizeImageBlock
-            imageSrc={stateObject.src}
+            imageSrc={stateObject.src ? stateObject.src : url!}
```

Comment:
The initial state for src is an empty string, so ternary operator is needed over `??` here and below.